corehq/apps/app_manager/views/multimedia.py

```diff
@@ -44,3 +47,18 @@ def get_multimedia_sizes(request, domain, app_id):
             mm_sizes.items()
         }
     return JsonResponse(mm_sizes)
+
+
+@require_deploy_apps
+@quickcache(['domain', 'app_id', 'other_build_id'], timeout=60 * 60)
+def compare_multimedia_sizes(request, domain, app_id, other_build_id):
```

Comment:
You could DRY this up a bit with `get_multimedia_sizes` above.

Reply:
https://github.com/dimagi/commcare-hq/pull/25868/commits/0d61ececc9c83b8cf5acf18e1f2b0717d03f2401